.github/scripts/download_assets.sh

```diff
@@ -1,46 +1,39 @@
 #!/bin/bash
 set -eu -o pipefail
 
-realpath() {
-    [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
-}
-
-# path to the directory where this file is
-SELF_DIR_PATH="$(dirname "$0")"
-
-# a parameter for the destination of the assets can be passed.
-# the default is the directory assets, we assume the script is in .github/scripts/
-DATA_DIR="${1:-$SELF_DIR_PATH/../../assets}"
-DATA_DIR=`realpath $DATA_DIR`
-
```

Comment:
:wrench: 
This code was ensuring that the assets are always downloaded in the `assets` directory at the top level of the project. With this change, they will be downloaded in the `../../assets` directory depending from where I run the script.